exercises/nucleotide-count/src/test/java/NucleotideCounterTest.java

```diff
@@ -35,68 +28,32 @@ public void testEmptyDnaStringHasNoNucleotides() {
 
     @Ignore("Remove to run test")
     @Test
-    public void testRepetitiveCytosineGetsCounted() {
-        NucleotideCounter nucleotideCounter = new NucleotideCounter("CCCCC");
-        assertThat(nucleotideCounter.count('C'), is(5));
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void testRepetitiveSequenceWithOnlyGuanine() {
-        NucleotideCounter nucleotideCounter = new NucleotideCounter("GGGGGGGG");
+    public void testDnaStringHasOneNucleotide() {
+        NucleotideCounter nucleotideCounter = new NucleotideCounter("G");
         Map<Character, Integer> counts = nucleotideCounter.nucleotideCounts();
         assertThat(counts.size(), is(4));
         assertThat(counts, allOf(
                 hasEntry('A', 0),
                 hasEntry('C', 0),
-                hasEntry('G', 8),
+                hasEntry('G', 1),
                 hasEntry('T', 0)
         ));
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void testCountsOnlyThymine() {
-        NucleotideCounter nucleotideCounter = new NucleotideCounter("GGGGGTAACCCGG");
-        assertThat(nucleotideCounter.count('T'), is(1));
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void testCountsANucleotideOnlyOnce() {
-        NucleotideCounter nucleotideCounter = new NucleotideCounter("CGATTGGG");
-        nucleotideCounter.count('T');
-        assertThat(nucleotideCounter.count('T'), is(2));
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void testDnaCountsDoNotChangeAfterCountingAdenine() {
-        NucleotideCounter nucleotideCounter = new NucleotideCounter("GATTACA");
-        nucleotideCounter.count('A');
+    public void testRepetitiveSequenceWithOnlyGuanine() {
+        NucleotideCounter nucleotideCounter = new NucleotideCounter("GGGGGGG");
         Map<Character, Integer> counts = nucleotideCounter.nucleotideCounts();
-        assertThat(counts.size(), is(4));
-        assertThat(counts, allOf(
-                hasEntry('A', 3),
-                hasEntry('C', 1),
-                hasEntry('G', 1),
-                hasEntry('T', 2)
-        ));
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void testValidatesNucleotides() {
-        expectedException.expect(IllegalArgumentException.class);
-        NucleotideCounter nucleotideCounter = new NucleotideCounter("GACT");
-        nucleotideCounter.count('X');
+        assertThat(nucleotideCounter.count('G'), is(7));
```

Comment:
Is there a reason you have used `assertThat(nucleotideCounter.count(...` instead of `assertThat(counts, allOf(hasEntry(...` (the structure of the other tests)?